modules/core/src/test/java/org/apache/ignite/internal/processors/cache/IgniteAbstractDynamicCacheStartFailTest.java

```diff
@@ -772,4 +934,254 @@ public BrokenStoreFactory(boolean eOnAllNodes, String gridName) {
                 return null;
         }
     }
+
+    /** Failure MBean server*/
+    private class FailureMBeanServer implements MBeanServer {
+
+        /** */
+        private final MBeanServer origin;
+
+        /** Set of caches that must be failure. */
+        private final Set<String> caches = new HashSet<>();
+
+        /** Set of nodes that must be failure. */
+        private final Set<String> nodes = new HashSet<>();
+
+        /** */
+        private FailureMBeanServer(MBeanServer origin) {
+            this.origin = origin;
+        }
+
+        /** Add cache name to failure set. */
+        void cache(String cache) {
+            caches.add('\"' + cache + '\"');
+        }
+
+        /** Add node name to failure set. */
+        void node(String node) {
+            nodes.add(node);
+        }
+
+        /** Clear failure set of caches and set of nodes. */
+        void clear() {
+            caches.clear();
+            nodes.clear();
+        }
+
+        /** {@inheritDoc} */
+        @Override public ObjectInstance registerMBean(Object object, ObjectName name)

```

Comment:
Use abbreviations in variable names. `object` -> `obj`, `class` -> `cls`, `loader` -> `ldr`, `query` -> `qry`.

Reply:
done